XCTWebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -61,7 +66,7 @@ + (NSArray *)routes
       NSInteger elementID = [request.parameters[@"id"] integerValue];
       XCUIElement *element = [elementCache elementForIndex:elementID];
       id text;
-      if ([element elementType] == XCUIElementTypeStaticText) {
+      if ([element elementType] == XCUIElementTypeStaticText || [element elementType] == XCUIElementTypeButton) {
```

Comment:
Alternative is to create wdText property, but this is only command we need it for so I'll leave it as is.